src/appleseed/renderer/meta/tests/test_volume.cpp

```diff
@@ -71,22 +71,28 @@ using namespace foundation;
 using namespace renderer;
 using namespace std::placeholders;
 
-TEST_SUITE(Renderer_Modeling_PhaseFunction)
+TEST_SUITE(Renderer_Modeling_Volume)
 {
     struct Fixture
       : public TestFixtureBase
     {
+        Fixture()
+          : TestFixtureBase()
+        {
+            m_base_parameters = ParamArray().insert("phase_function_model", "henyey");
```

Comment:
I think the tests would be clearer if we would get rid of `m_base_parameters` and just repeat the phase function model.